src/ExchangeSharp/API/Exchanges/Coinbase/ExchangeCoinbaseAPI.cs

```diff
@@ -396,24 +396,25 @@ protected override async Task<ExchangeOrderResult> OnPlaceOrderAsync(ExchangeOrd
   		payload["side"] = order.IsBuy ? BUY : "SELL";
 
   		Dictionary<string, object> orderConfig = new Dictionary<string, object>();
+			var amount = order.ShouldRoundAmount ? order.RoundAmount().ToStringInvariant() : order.Amount.ToStringInvariant();
```

Comment:
We check the ShouldRound and then `RoundAmount` if needed

Reply:
This check happens already in `order.RoundAmount` so i moved back to using the method instead of this variable check